### PR TITLE
ARCH-867 - Adding the initial action for the Octo-like view

### PR DESCRIPTION
**Pull request by danielle-casella-adams**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,80 +1,112 @@
...
+        with:
+          github-token: ${{ secrets.BOT_TOKEN}}             # If a different token is used, update github-login with the corresponding account
+          github-login: 'my-bot'
+          environment: 'QA'
+          board-number: 2
+          ref: 'feature-branch-16'
+          ref-type: 'branch' 
+          deploy-status: ${{ steps.deploy-to-qa.outcome }}  # outcome is the result of the step before continue-on-error is applied
+          timezone: 'america/denver'
 ```
```
Comment (izep) [inline README.md:110]: I think i'd add to the example a "fail the build" step that returns the error from the deploy step if there was one.
Something like: [this](https://github.com/bc-swat/github-services-infrastructure/blob/31b9e9e46b9d80d44036b494cf39d54a7bb74ca9/.github/workflows/tf-actions-runners-auto-plan-and-comment-on-prs.yml#L91)

**Review by hpractv — DISMISSED** [review]
👍🏽

File: README.md (hunk 1 above)
Comment (danielle-casella-adams) [reply README.md:116]: I meant to do that and forgot.

**Review by izep — APPROVED** [review]

**Review by hpractv — APPROVED** [review]
👍🏽  x 2

**Pull request merged**